api/s2n.h

```diff
@@ -2986,12 +2994,14 @@ struct s2n_async_pkey_op;
  * - `S2N_ASYNC_PKEY_VALIDATION_FAST` - default behavior: s2n-tls will perform only the minimum validation required for safe use of the asyn pkey operation.
  * - `S2N_ASYNC_PKEY_VALIDATION_STRICT` - in addition to the previous checks, s2n-tls will also ensure that the signature created as a result of the async private key sign operation matches the public key on the connection.
  */
-typedef enum { S2N_ASYNC_PKEY_VALIDATION_FAST, S2N_ASYNC_PKEY_VALIDATION_STRICT } s2n_async_pkey_validation_mode;
+typedef enum { S2N_ASYNC_PKEY_VALIDATION_FAST,
+    S2N_ASYNC_PKEY_VALIDATION_STRICT } s2n_async_pkey_validation_mode;
 
 /**
  * The type of private key operation
  */
-typedef enum { S2N_ASYNC_DECRYPT, S2N_ASYNC_SIGN } s2n_async_pkey_op_type;
+typedef enum { S2N_ASYNC_DECRYPT,
+    S2N_ASYNC_SIGN } s2n_async_pkey_op_type;
```

Comment:
What happened to these enums?

Reply:
;( -- I don't know! Taking a look

Reply:
So not quite sure what was going on here but I found a really nice solution -- move all enums to be multi-line. It takes a bit more vertical height but everything is very uniform.